tools/block-generator/generator/make_transactions.go

```diff
@@ -158,7 +158,7 @@ func (g *generator) makeAppCreateTxn(kind appKind, sender basics.Address, round,
 
 	// also group in a pay txn to fund the app
 	pstFee := uint64(1_000)
-	pstAmt := uint64(1_000_000)
+	pstAmt := uint64(2_000_000)
```

Comment:
apparently, the checks are more stringent now that apply data is being provided.